lib/core/src/lib/form/components/widgets/core/form.model.spec.ts

```diff
@@ -284,18 +284,22 @@ describe('FormModel', () => {
         expect(form.outcomes[1].isSystem).toBeFalsy();
     });
 
-    it('should raise validation event when validating form', (done) => {
+    it('should raise validation event when validating form', () => {
         const form = new FormModel({}, null, false, formService);
 
-        formService.validateForm.subscribe(() => done());
+        formService.validateForm.subscribe((validateFormEvent) =>
+            expect(validateFormEvent).toBeTruthy()
```

Comment:
this expect may not wait for the test end

Reply:
this test is synchronous, so we don't need to wait for anything here, we just checking if the validation event was raised. before we were using done method to check if it will reach subscription, instead of that we can expect some validation event.